STYLE_GUIDE.md

```diff
@@ -123,7 +118,7 @@ Kedro-Viz has both light and dark themes, and every component must be styled app
 
 ![Kedro-Viz Color Styleguide](./.github/img/kedro_colours.png)
 
-The themes are set using the same theme classes that Kedro-UI uses - `kui-theme--light` and `kui-theme--dark` - applied on the top-level container element. To simplify code and avoid increasing CSS specificity, we use CSS custom properties to handle theme colours. We recommend that you use these too where possible - they are defined in the App component.
+The themes are set using two classes - `kui-theme--light` and `kui-theme--dark` - applied on the top-level container element. To simplify code and avoid increasing CSS specificity, we use CSS custom properties to handle theme colours. We recommend that you use these too where possible - they are defined in the App component.
```

Comment:
I recall this being discussed in a previous PR, we should remove the naming reference of `kui-` in the theme classes as the last piece of the puzzle to complete the Kedro-UI migration work - this could be scoped out in a seperate PR.

Reply:
Completely agree. Good call.